.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: node_js
+notifications:
+  email: false
+node_js:
+  - '6'
+install: npm install
```

Comment:
They're same, you can just write single command on the same line, but multiple should be a list.

initially `script` consisted of 2 commands, that's why, I can make both single line.